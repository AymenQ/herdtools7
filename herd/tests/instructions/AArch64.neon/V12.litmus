AArch64 V12
(* Tests load / store immediate unscaled, no offset *)
{ 
0:X0=x; 0:X2=y;
}

 P0                  ;
 MOV X1, #1          ;
 STR X1, [X0, #8]    ;
 LDUR B0, [X0, #8]   ;
 STUR B0, [X2]       ;
 forall (y=1)
 