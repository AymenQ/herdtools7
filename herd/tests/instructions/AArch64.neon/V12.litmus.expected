Test V12 Required
States 1
y=1;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall (y=1)
Observation V12 Always 1 0
Hash=2f917ee63a8382b1a2077f1d9871ff03

